bundles/AdminBundle/Controller/Admin/TagsController.php

```diff
@@ -125,9 +126,30 @@ public function treeGetChildrenByIdAction(Request $request)
         }
         $tagList->setOrderKey('name');
 
+        if(!empty($request->get('filter'))) {
+            $filterIds = [0];
+            $filterTagList = new Tag\Listing();
+            $filterTagList->setCondition('`name` LIKE "%'.$request->get('filter').'%"');
```

Comment:
SQL injection possible. Please properly quote the filter.
Example: http://pimcore-demo-basic.pim.zone/admin/tags/tree-get-children-by-id?_dc=1580295370027&showSelection=false&assignmentCId=&assignmentCType=&node=0&filter=%22;%20DROP%20TABLE%20objects;%20SELECT%20*%20FROM%20users%20WHERE%20name%20LIKE%20%22

will drop the objects table